modules/azure/compute_test.go

```diff
@@ -13,18 +13,63 @@ import (
 
 /*
 The below tests are currently stubbed out, with the expectation that they will throw errors.
-If/when CRUD methods are introduced for Azure Virtual Machines, these tests can be extended
-(see AWS S3 tests for reference).
+If/when CRUD methods are introduced for Azure Virtual Machines, these tests can be extended.
 */
 
-func TestGetTagsForVirtualMachineE(t *testing.T) {
+func TestGetVirtualMachineE(t *testing.T) {
 	t.Parallel()
 
 	vmName := ""
 	rgName := ""
 	subID := ""
 
-	_, err := GetTagsForVirtualMachineE(t, vmName, rgName, subID)
+	_, err := GetVirtualMachineE(vmName, rgName, subID)
+
+	require.Error(t, err)
+}
+
+func TestGetVirtualMachineInstanceE(t *testing.T) {
+	t.Parallel()
+
+	vmName := ""
+	rgName := ""
+	subID := ""
+
+	_, err := GetVirtualMachineInstanceE(vmName, rgName, subID)
+
+	require.Error(t, err)
+}
+
+func TestListVirtualMachinesForResourceGroupE(t *testing.T) {
```

Comment:
Are these all unit tests just checking the methods handle invalid params?

Reply:
It has been helpful as the library changed with number of params and names updated as well.